src/main/index.ts

```diff
@@ -161,6 +166,27 @@ function postLoadConfig(): void {
   });
 }
 
+/**
+ * Save all the messages from the console log to a .log file
+ */
+function saveLog(messages: string): void {
+  if (!mainWindow) return;
+
+  const fileName = moment().format('[GCS Log] YYYY-MM-DD [at] h.mm.ss A');
```

Comment:
Possibly change from using periods (.) to colons (:) for enhanced readability.

Reply:
The reason they're periods is because I wanted to model these files to match those generated when you take a screenshot in macOS. The filename when you take a screenshot is as follows:

`Screen Shot 2020-03-17 at 3.27.56 PM`